niworkflows/reports/core.py

```diff
@@ -462,10 +462,7 @@ def _process_orderings(orderings, layout):
             {value[idx] for value in all_value_combos} for idx in range(len(orderings))
         ]
         # if all values are None for an entity, we do not want to keep that entity
-        keep_idx = [
-            not ((len(val_set) == 1 and None in val_set) or not val_set)
-            for val_set in unique_values
-        ]
+        keep_idx = [val_set not in (set(), {None}) for val_set in unique_values]
```

Comment:
I find this more readable:
```suggestion
        keep_idx = [vla_set and val_set != {None} for val_set in unique_values]
```

Reply:
But this one is definitely shorter:
```suggestion
        keep_idx = [val_set not in ({}, {None}) for val_set in unique_values]
```

Reply:
That's a dict literal.